project/JdkOptions.scala

```diff
@@ -48,7 +48,9 @@ object JdkOptions extends AutoPlugin {
       targetSystemJdk,
       jdk8home,
       fullJavaHomes,
-      Seq(if (scalaVersion.startsWith("3.")) "-Xtarget:8" else "-target:jvm-1.8"),
+      if (scalaVersion.startsWith("3.")) Seq("-Xtarget:8")
+      else if (scalaVersion.startsWith("2.13.")) Seq("-release", "8")
```

Comment:
where can I read about the reason for this change?
It's somewhat scary given the below comment "-release 8' is not enough, for some reason we need the 8 rt.jar"

Reply:
In https://github.com/scala/scala/pull/9982, I have compiled it with jdk 11 and checked the bytecode, is there anything I missed out?

Reply:
Thanks. All good!